var/spack/repos/builtin/packages/flang/package.py

```diff
@@ -35,10 +35,16 @@ class Flang(CMakePackage):
     version('develop', git='https://github.com/flang-compiler/flang', branch='master')
 
     depends_on(
-        "llvm+clang",
-        patches=patch('https://github.com/llvm-mirror/clang/pull/33.diff',
+            "llvm+clang@4.0.1:5.0",
```

Comment:
My understanding is that you have to patch each specific version of LLVM, otherwise flang will fail to build. Previously only 4.0.1 was patched, now also 5.0.0. As such, instead of listing a range @4.0.1:5.0, which suggests all the versions between 4.0.1 to 5.0.0, I would rather list those specific versions of LLVM that are supported, i.e. something like `@4.0.1,5.0.0`, or whatever the syntax is. 

The reason I am asking is that it takes a long time to build and we know it's going to fail, unless the LLVM versions are either 4.0.1 or 5.0.0.

If LLVM releases 4.0.2 and it is added to spack, and this flang package is not updated, then a build failure will happen.

Reply:
Good point - fixed!